test/functional/web/helpers.js

```diff
@@ -3,7 +3,9 @@ import { HOST, PORT } from '../helpers/session';
 import { util } from 'appium-support';
 
 
-const BASE_END_POINT = `http://${process.env.REAL_DEVICE ? util.localIp() : HOST}:${PORT}`;
+const LOCAL_BASE_END_POINT = `http://${process.env.REAL_DEVICE ? util.localIp() : HOST}:${PORT}`;
+const REMOTE_BASE_END_POINT = `https://quizzical-tesla-615650.netlify.com`;
```

Comment:
What is this?

Reply:
I needed to host the guinea pig site so I used netlify which has free static hosting. The Sauce labs static site was outdated

Reply:
If we just changed it so that the HOST and PORT for the base endpoint are not the host/port for the command executor, but rather for the Appium server, then it would work for hitting what we serve.

This seems like a recipe for headaches, keeping things in sync when, inevitably, they are changed.

Reply:
I don't think on-demand hosts that static site. If we do want to keep it synchronized its probably best that we add a publish script to base driver that deploys the static site

Reply:
On-demand does not. Appium does, though. So, in Sauce the simulator is running on the same place as the Appium server, and whatever the address/port is for that, we can hit that. It is 127.0.0.1 and 4443 on Sauce.